src/reconcile/rec_visibility.c

```diff
@@ -726,6 +742,15 @@ __wt_rec_upd_select(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_INSERT *ins, W
         supd_restore = F_ISSET(r, WT_REC_EVICT) &&
           (has_newer_updates || F_ISSET(S2C(session), WT_CONN_IN_MEMORY));
 
+        /*
+         * The total update size only contains uncommitted updates. Add the size for the rest of the
+         * chain.
+         */
```

Comment:
Add a FIXME comment with the ticket number that you created to deal with the cache stuck issues when we are not considering the updates that are written to the disk.